packages/devtools_app/lib/src/code_size/code_size_controller.dart

```diff
@@ -59,9 +60,7 @@ class CodeSizeController {
       treemapNodeSize = treeJson['value'] ?? 0;
     }
 
-    final root = TreemapNode(name: treemapNodeName, byteSize: treemapNodeSize);
-    root.addAllChildren(treemapNodeChildren);
-    return root;
+    return TreemapNode(name: treemapNodeName, byteSize: treemapNodeSize)..addAllChildren(treemapNodeChildren);
```

Comment:
if you run dartfmt this will probably format with a line break

Reply:
Done